packages/shared/lib/services/sync/sync.service.ts

```diff
@@ -112,30 +112,9 @@ export async function setFrequency(id: string, frequency: string | null): Promis
 
 /**
  * Set Last Sync Date
- * @desc if passed a valid date set the sync date, however if
- * we don't want to override the sync make sure it is null
- * before we set it
- *
- * This is due to the fact that users can set the last sync date
- * during the integration script so we don't want to override what they
- * set in the script
  */
-export const setLastSyncDate = async (id: string, tempDate: Date): Promise<boolean> => {
-    const date = typeof tempDate === 'string' ? new Date(tempDate) : tempDate;
-
-    if (isNaN(date.getTime())) {
-        return false;
-    }
```

Comment:
Forgot to remove this unnecessary logic now that we only passes Date in a controlled way